src/howard/howard/issuers/invoice.py

```diff
@@ -0,0 +1,66 @@
+""""Invoice issuer"""
+
+from pathlib import Path
+
+from pydantic import BaseModel
+
+from marion.issuers.base import AbstractDocument
+
+
+class Customer(BaseModel):
+    """Customer pydantic model"""
+
+    name: str
+    address: str
+
+
+class Product(BaseModel):
+    """Product pydantic model"""
+
+    name: str
+    description: str
+    total: str
+    subtotal: str
+    vat_mount: str
+    vat: str
```

Comment:
Can you switch the four last fields into Decimal?
```suggestion
    vat: decimal.Decimal
```

Reply:
+1 for validating that amounts are Decimals with 2 digits (for money) as we don't want to print an invoice with $34.5 instead of $3.50.

Then the currency should be a separate field, but can be of type `str` because maintaining a enum is a little overkill and we should let the calling application decide if they want to display `€` or `EUR` or `euros` or `eur`, etc. The position of the currency relative to the amount can be changed in the template so that's :+1: